nomos-services/data-availability/network/src/lib.rs

```diff
@@ -272,6 +292,26 @@ where
                         "client hung up before a subscription handle could be established"
                     );
                 }),
+            DaNetworkMsg::SubnetworksAtBlock {
+                block_number,
+                sender,
+            } => {
+                if let Some(membership) = membership_storage.get_historic_membership(block_number) {
+                    let assignations = membership.subnetworks();
+                    sender.send(assignations).unwrap_or_else(|_| {
+                        tracing::warn!(
+                            "client hung up before a subnetwork assignations handle could be established"
+                        );
```

Comment:
This should be an error probably (?), it means that the other service was terminated. Maybe not and a warning is enough.

Reply:
I agree. I was thinking about this as well and I am not sure. Maybe we can solve it when we implement a logic that uses this message, it should be more clear then. It should be straightforward to introduce an error

Reply:
Add a todo on what would be needed, so when coming back to the code we relate.